resources/WorkRecordResource.json

```diff
@@ -1,44 +1,353 @@
 {
-    "description": "Abstract class that represents a work record - an activity that has been carried out.",
-    "type": "object",
-    "required": [
-        "operation",
-        "status"
-    ],
-    "allOf": [
-        {
-            "$ref": "../types/ObservationType.json"
-        }
-    ],
-    "properties": {
-        "operation":{
-            "type":"string",
-            "description":"Constant value in any concrete class for the operation type"
-        },
-        "status":{
-            "$ref":"../enums/WorkItemStatusType.json",
-            "description": "The status of the work record."
+  "description": "Abstract class that represents a work record - an activity that has been carried out.",
+  "type": "object",
+  "required": [
+    "operation",
+    "status"
+  ],
+  "allOf": [
+    {
+      "$ref": "../types/ObservationType.json"
+    }
+  ],
+  "properties": {
+    "operation": {
+      "type": "string",
+      "description": "Constant value in any concrete class for the operation type"
+    },
+    "status": {
+      "$ref": "../enums/WorkItemStatusType.json",
+      "description": "The status of the work record."
+    },
+    "priority": {
+      "$ref": "../enums/WorkItemPriorityType.json",
+      "description": "The priority of the work record."
+    },
+    "workedArea": {
+      "$ref": "../types/AreaMeasureType.json",
+      "description": "Area worked by the operation"
+    },
+    "nonOverlapWorkedArea": {
+      "$ref": "../types/AreaMeasureType.json",
+      "description": "Area worked by the operation, excluding overlaps"
+    },
+    "loggedOperations": {
+      "description": "array of operations completed in this work record",
+      "type": "array",
+      "items": {
+        "$ref": "../resources/OperationRecordResource.json"
+      }
+    }
+  },
+  "examples": [
+    {
+      "resourceType": "/crop/work-record",
```

Comment:
Not sure that the ResourceType examples are correct. Normally they should reflect the Geospatial JSON file name. So WorkRecordResource, OperationRecordResource. Otherwise most code generated using openapi-generator will fail.

Reply:
Changed Resource Types